src/libstore/local-store.md

```diff
@@ -0,0 +1,40 @@
+R"(
+
+**Store URL format**: `local`, *root*
+
+This store type accesses a Nix store in the local filesystem directly
+(i.e. not via the Nix daemon). *root* is an absolute path that denotes
+the "root" of the filesystem; other directories such as the Nix store
+directory (as denoted by the `store` setting) are interpreted relative
+to *root*. The store pseudo-URL `local` denotes a store that uses `/`
+as its root directory.
+
+A store that uses a *root* other than `/` is called a *chroot
+store*. With such stores, the store directory is "logically" still
+`/nix/store`, so programs stored in them can only be built and
+executed by `chroot`-ing into *root*. Chroot stores only support
+building and running on Linux when mount and user namespaces are
+enabled.
+
+For example, the following uses `/tmp/root` as the chroot environment
+to build or download `nixpkgs#hello` and then execute it:
+
+```console
+# nix run --store /tmp/root nixpkgs#hello
+Hello, world!
+```
```

Comment:
I think we should make a definitive decision on whether and how to document experimental features in the stable parts of the manual. [Similar situation in the hacking guide.](https://github.com/NixOS/nix/pull/8082/files#diff-9987ae82abaca81e14ce3fbd0b7a55c1ea76f911148d632c5c6a97d889ce89eeR400)

While this, for now, will only show in `nix` command docs, there is no reason not to have it everywhere it matters, since that stuff is already stable.